keras/preprocessing/image.py

```diff
@@ -963,6 +963,9 @@ def _recursive_list(subpath):
         for fname in files:
             is_valid = False
             for extension in white_list_formats:
+                if fname.lower().endswith('.tiff'):
+                    warnings.warn('Using \'.tiff\' files with multiple bands will cause distortion.'
+                                  'Please verify your output.')
```

Comment:
Do we really need this warning?

Reply:
IMO yes or we shouldn't accept tiff at all. 
We'll get complaints that some .tiff are not handled correctly.

Reply:
Ok then. The message is missing a space for the period, please fix

Reply:
Sorry about that. Fixed.